github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -2589,3 +2589,81 @@ periodics:
         privileged: true
     nodeSelector:
       type: bare-metal-external
+- annotations:
+    testgrid-dashboards: kubevirt-periodics
+    testgrid-days-of-results: "60"
+  cluster: kubevirt-prow-control-plane
```

Comment:
sorry for being late to the party, but does control-plane cluster have enough resources to run the tests in question?

/cc @brianmcarey @jschintag

Reply:
It at least ran the rehearsal. So it can run it, but i don't know what resources it has or what other tasks it needs to run with them.